docs/separate_production_and_staging_inventory.md

```diff
@@ -0,0 +1,74 @@
+# Separate production and staging inventory
+
+You can keep your production environment separate from all your other environments by using parted inventory files or even directories for each environment.
+
+This makes the inventory files clearer and they don't get too cluttered over time. You can also combine multiple inventory source types in an inventory directory. This can be useful for combining static and dynamic hosts and managing them as one inventory.
+
+## Bad
+
+Folder structure:
+
+```bash
+/path/to/your/ansible
+├── ansible.cfg
+├── group_vars
+├── host_vars
+├── inventory_file
+├── playbooks
+├── roles
+```
+
+File structure:
+
+```yaml
+[example_test_web]
+example_test_web01
+
+[example_live_web]
+example_live_web01
+
+[example_live:children]
+example_live_web
+
+[example_test:children]
+example_test_web
+
+```
+
+## Good
+
+Folder structure
+
+```bash
+/path/to/your/ansible
+├── ansible.cfg
+├── group_vars
```

Comment:
Should the group_vars and host_vars also live in the inventory-directories?

```
/path/to/your/ansible
├── ansible.cfg
├── group_vars
├── host_vars
├── inventory
│   ├── live
 |            |── group_vars
 |            |── host_vars
│   └── test
├── playbooks
├── roles
```

Reply:
Should they? I don't know if you should nest them in an inventory folder. @schurzi what do you think about this?

Reply:
It's definitely possible. This way, you separate the variables for the environments, too. Of course this also means duplication (e.g. in `all.yml`).

Reply:
Makes sense to me. I've added your suggestion to the documentation.